main/guides/js-programming/notifiers.md

```diff
@@ -130,16 +129,17 @@ The `updater` and `publication` both have the same three methods:
     signalling that the monitored object hit an error condition.
 
 The `subscription` and `publication` both have this method:
```

Comment:
Neither do. This method is only on notifier.

Reply:
So notifiers and subscriptions can both be consumed with `for await`, but only notifiers can be consumed with `.getUpdateSince`?
```suggestion
The `notifier` has an additional method that the `subscription` does not:
```

Reply:
yes

Reply:
They can also both be consumed with `observeIteration`. We should move towards considering `getUpdateSince` to be an implementation detail, and recommend that consumption happen either by `for await` or `observeIteration`.